websites.json

```diff
@@ -106,5 +106,12 @@
         "about": "I do things I find cool and talk about it",
         "url": "https://garfunkles.space",
         "owner": "https://github.com/NamesCode"
+    },
+    {
+        "name": "lighttigerXIV",
+        "slug": "lighttigerxiv",
+        "about": "lighttigerXIV personal website",
+        "url": "https://lighttigerxiv.vercel.app",
```

Comment:
could you make this URL point to the contact page. Then we are good to go!

Reply:
@isabelroses it should be good now